### DISCUSSION TITLE
FN4 Review

### DISCUSSION THREAD

**Discussion by kathypinzon** [TeamBasedInquiryLearning/precalculus#125] [General]
Section FN4: https://teambasedinquirylearning.github.io/precalculus/2024/FN4.html
Author: @kathypinzon 
Reviewer @tdegeorge

**Comment by kathypinzon** [comment]
p+=text(“$f(x)+1$”, (1, 4), fontsize=18, color='black') - fix labels of graphs in math mode

**Comment by tdegeorge** [comment]
Since we are fixing these....should we go back and put answers in?

Also - would gridlines be good for these graphs?

**Reply by kathypinzon** [reply]
they have answers... maybe you weren't looking at the right version?
Added gridlines.

**Comment by tdegeorge** [comment]
For all the definitions, can we take out the "y=" part? Can we just say that the transformed function, g(x), is equal to....

**Comment by tdegeorge** [comment]
Remark 2.4.17: There is a typo - "reflection"

**Comment by tdegeorge** [comment]
Activity 2.4.18: I don't think the answer is there....I don't see a horizontal shift in the answers

**Comment by tdegeorge** [comment]
Should we maybe start by asking students to look at y-values of the graph before getting to activities 2.4.19 and 2.4.20? It might be helpful if they were to "explore" a little more (maybe just the vertical stretch) how the y-values are changed and then move on to the wording of "stretching by a factor of 2"

**Reply by kathypinzon** [reply]
adding a part before the vertical stretch question asking how the y-value at x=1 compares.

**Comment by tdegeorge** [comment]
Definition 2.4.22 - typo - the second "vertical" has two c's in it

**Comment by tdegeorge** [comment]
Activity 2.4.25: Would it be helpful to ask students to graph the parent function first?

**Comment by tdegeorge** [comment]
Activity 2.4.26: Typo in part c - "representd"

**Comment by sheppardw** [comment]
Activity 2.4.10 
Do you want to say that they can choose more than one answer or just leave it open for them to figure out on their own?

**Comment by sheppardw** [comment]
Remark 2.4.17.
Typo in the word "reflection"

**Comment by sheppardw** [comment]
Remark 2.4.17.
Can you go into more explanation with the order of operations here? Or give another example maybe?

**Comment by sheppardw** [comment]
Activity 2.4.18
Maybe you could ask them to order the transformations to emphasize Remark 2.4.17?

**Comment by sheppardw** [comment]
Definition 2.4.22. 
typo in the second word vertical

**Comment by sheppardw** [comment]
Remark 2.4.24. 
For these graphs, can we put the graph names in math type? I don't know if we can put arrows on the ends of the graphs also?

**Comment by sheppardw** [comment]
Activity 2.4.26. 
c) typo on represent

**Comment by sheppardw** [comment]
Do we want to give them examples with the a value being between 0 and 1 and have them make their own conclusion versus telling them what the difference is for the vertical and horizontal stretches and compressions?
